Modules/xSharePoint/DSCResources/MSFT_xSPFeature/MSFT_xSPFeature.psm1

```diff
@@ -47,23 +49,55 @@ function Set-TargetResource
         [parameter(Mandatory = $true)]  [System.String] $Url,
         [parameter(Mandatory = $true)]  [ValidateSet("Farm","WebApplication","Site","Web")] [System.String] $FeatureScope,
         [parameter(Mandatory = $false)] [System.Management.Automation.PSCredential] $InstallAccount,
-        [parameter(Mandatory = $true)]  [ValidateSet("Present","Absent")] [System.String] $Ensure
+        [parameter(Mandatory = $true)]  [ValidateSet("Present","Absent")] [System.String] $Ensure,
+        [parameter(Mandatory = $false)] [System.String] $Version
     )
 
     $result = Invoke-xSharePointCommand -Credential $InstallAccount -Arguments $PSBoundParameters -ScriptBlock {
         $params = $args[0]
         
 
-        $runParams = @{}
-        $runParams.Add("Identity", $params.Name)
+        $runParams = @{ 
+            Identity = $params.Name 
+            Verbose = $false
```

Comment:
Can we take out the verbose = $false line. The Verbose logging is something we have aimed to use where it happens out of the box (so not specifying it false or true either way) so the output is consistent with what happens out of the box.